dev-tools/generate_notice.py

```diff
@@ -102,6 +134,11 @@ def create_notice(filename, beat, copyright, vendor_dirs):
 
         if 'vendor' in dirs:
             vendor_dirs.append(os.path.join(root, 'vendor'))
+            dirs.remove('vendor')   # don't walk down into sub-vendors
```

Comment:
Ok, I was thinking of vendor directories inside the vendor dirs. I remember seeing that somewhere.